posts/2023-09-05-Increasing-Apple-Version-Requirements.md

```diff
@@ -0,0 +1,79 @@
+
+---
+layout: post
+title: "Increasing the minimum supported Apple platform versions"
+author: BlackHoleFox
+description: "Modernizing and improving Apple platform support for Rust"
+---
+
+As of Rust <CURRENT_RUST_VERSION/> (to be released on <DATE/>), the minimum version of Apple's platforms (iOS, macOS, and tvOS) that the Rust toolchain supports will be [increased](https://github.com/rust-lang/rust/pull/104385) to newer minimums. These changes affect both the Rust compiler itself (`rustc`), other host tooling, and most importantly, the standard library and any binaries produced that use it. With these changes in place, any binaries produced will stop loading on older versions or exhibit other, unspecified, behavior.
```

Comment:
Adding a callout for this placeholder which needs filled in once the actual PR with the changes merges and we know what release its scheduled for.

Reply:
The PR merged so it should be in 1.74 if all goes well, updated.